zaas-service/src/test/resources/zaas-messages.yml

```
@@ -322,7 +301,7 @@ messages:
       type: ERROR
       text: "There was an error while reading webfinger configuration"
       reason: "Webfinger provider contains incorrect configuration."
-      action: "Contact the administrator to validate webfinger configuration in gateway service."
+      action: "Contact the administrator to validate webfinger configuration in the ZAAS."
```

```suggestion
      action: "Contact the administrator to validate webfinger configuration in ZAAS."
```
Or just the full name Contact the administrator to validate webfinger configuration in the Zowe Authentication and Authorization Service.